core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -346,7 +346,9 @@ fun String.tr(): String {
     }
 
     if (contains('{')) { // Translating partial sentences
-        return curlyBraceRegex.replace(this) { it.groups[1]!!.value.tr() }
+        // ..but not if they're contained in square brackets?
```

Comment:
When did this switch around? The order should be conditionals - placeholders - templating
< [ {

Reply:
Wow, I didn't catch that even though I did code in the routine long before... ... Tada ... #6161 did it. Will draft until I can really investigate why that was, what for, and what a revert would cost / side effects and so on... Which may well not be until ~10 days from now.